### PR TITLE
docs: added new locale string and release notes

### PR DESCRIPTION
**Pull request by saksham-gt**
## Description

- [x] Added release notes for v0.7.9

### CODE REVIEW

File: docs/flutter/meeting-locale.mdx [hunk 1]
```diff
@@ -31,68 +31,84 @@ Navigator.push(context, MaterialPageRoute(builder: (context) => uikit),);
...
+| "you"                                                             | "_\<locale-word>_"   |
+| "turnOffVideo"                                                    | "_\<locale-word>_"   |
+| "videoAlreadyOff"                                                 | "_\<locale-word>_"   |
+| "dyte"                                                            | "_\<locale-word>_"   |
+| "back"                                                            | "_\<locale-word>_"   |
+| "waitingForTheHostToLetYouIn"                                     | "_\<locale-word>_"   |
+| "shareScreen"                                                     | "_\<locale-word>_"   |
+| "stopSharing"                                                     | "_\<locale-word>_"   |
 
 3. All set, run the app and view the UI Kit in the desired language!
```
Comment (swapnilmadavi) [inline docs/flutter/meeting-locale.mdx:113]: (suggestion): The 3rd point feels a bit out-of-flow due to the big table between the 2nd & 3rd point. 
How about moving the 3rd point above and keeping the "strings table" under a separate header at the bottom of the page?

**Review by swapnilmadavi — CHANGES_REQUESTED** [review]

File: static/release-notes/flutter-ui-kit.json [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+    "fixes": ["Introduced newer strings for custom locale."]
+  },
+  {
+    "version": "0.7.8",
+    "createdAt": 1721673000,
+    "fixes": [
+      "Make listeners list safe from modification while callbacks in progress",
+      "Participant video sync in `DyteParticipantTile` component."
+    ]
+  },
```
Comment (swapnilmadavi) [inline static/release-notes/flutter-ui-kit.json:14]: (doubt): Same changelog as [v0.7.7](https://454d96b7.dyte-docs.pages.dev/flutter/release-notes#0.7.7)?

File: docs/flutter/meeting-locale.mdx [hunk 3]
```diff
@@ -31,68 +31,84 @@ Navigator.push(context, MaterialPageRoute(builder: (context) => uikit),);
...
-| "you"                             | "_\<locale-word>_" |
-| "turnOffVideo"                    | "_\<locale-word>_" |
-| "videoAlreadyOff"                 | "_\<locale-word>_" |
-| "dyte"                            | "_\<locale-word>_" |
-| "back"                            | "_\<locale-word>_" |
-| "waitingForTheHostToLetYouIn"     | "_\<locale-word>_" |
-| "shareScreen"                     | "_\<locale-word>_" |
-| "stopSharing"                     | "_\<locale-word>_" |
+| `key` | `value` |
+| ----- | ------- |
```
[the remark this answers is not in the document]
Comment (saksham-gt) [reply docs/flutter/meeting-locale.mdx]: done.

File: static/release-notes/flutter-ui-kit.json (hunk 2 above)
Comment (saksham-gt) [reply static/release-notes/flutter-ui-kit.json]: changed.

File: docs/flutter/meeting-locale.mdx (hunk 1 above)
Comment (saksham-gt) [reply docs/flutter/meeting-locale.mdx:110]: Sure, we can do that, it justs mentions that to run the app. Don't think it'll make much of a difference.

**Review by swapnilmadavi — APPROVED** [review]
LGTM! 👍

**Pull request merged**